modules/openthread/Kconfig.thread

```
@@ -87,11 +87,17 @@ config OPENTHREAD_CSL_RECEIVE_TIME_AHEAD
 	int "CSL receiver wake up margin in microseconds"
 	default 5000
 
-config OPENTHREAD_CSL_MIN_RECEIVE_ON
-	int "Minimum CSL receive window"
-	default 5696
+config OPENTHREAD_MIN_RECEIVE_ON_AHEAD
+	int "Minimum receiving time before start of MHR"
+	default 192
 	help
-	  The minimum CSL receive window (in microseconds) required to receive a full IEEE 802.15.4 frame
+	  The minimum time that radio has to be in receive mode before the start of the MHR (microseconds).
```

```suggestion
	  The minimum time  (microseconds) that radio has to be in receive mode before the start of the MHR
	  to receive an IEEE 802.15.4 frame.
```

Clarify this in all places?